modules/services/pass-secret-service.nix

```
@@ -0,0 +1,24 @@
+{ pkgs, config, lib, ... }:
+let serviceCfg = config.services.pass-secret-service;
+in with lib; {
+  meta.maintainers = [ maintainers.cab404 ];
+  options.services.pass-secret-service = {
+    enable = mkEnableOption "Pass libsecret service";
+  };
+  config = mkIf serviceCfg.enable {
+    assertions = [{
+      assertion = config.programs.password-store.enable;
+      message = "The 'services.pass-secret-service' module requires"
+        + " 'programs.password-store.enable = true'.";
+    }];
+
+    systemd.user.services.pass-secret-service = {
+      Unit = { Description = "Pass libsecret service"; };
+      Service = {
+        # pass-secret-service doesn't use environment variables for some reason.
+        ExecStart =
+          "${pkgs.pass-secret-service}/bin/pass_secret_service --path $PASSWORD_STORE_DIR";
```

Not an expert, but does it make sense to use `config.programs.password-store.settings.PASSWORD_STORE_DIR` directly instead of relying on that being in the environment when the service starts?

```suggestion
          "${pkgs.pass-secret-service}/bin/pass_secret_service --path ${config.programs.password-store.settings.PASSWORD_STORE_DIR}";
```

---

Good catch 👍

---

Maybe we should do the same for pass.rofi then)